metadata_mapper/lambda_function.py

```diff
@@ -56,7 +56,8 @@ def parse_enrichment_url(enrichment_url):
 
 
 def run_enrichments(records, collection, enrichment_set, page_filename):
-    for enrichment_url in collection.get(enrichment_set, []):
+    enrichment_urls = collection.get(enrichment_set) or []
```

Comment:
You've done this in several places, replacing `collection.get(some_value, [])` with `collection.get(some_value) or []`. Is there a difference in behavior between those two? It isn't obvious to me what that is.  No issue with it, just wondering.

Reply:
@amywieliczka @bibliotechy so yeah, it turns out that if you explicitly set the value to `None` then this is what happens:

```
Python 3.9.0 (default, May 23 2023, 15:28:21)
[Clang 13.0.0 (clang-1300.0.27.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> mydict = {"foo": None}
>>> for x in mydict.get("foo", []):
...    print(x)
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not iterable
```

Whereas:
```
>>> mydict.get("foo") or []
[]
```